projects/js-packages/react-data-sync-client/jsconfig.json

```diff
@@ -0,0 +1,8 @@
+// Config for a JS-using project. If using typescript, rename this to "tsconfig.json".
```

Comment:
Since it looks like you're planning to use typescript, you should probably do this.

Oh, you already added a separate tsconfig.json. In that case you should just delete this file.